node/core/pvf/worker/src/common.rs

```diff
@@ -68,13 +65,12 @@ pub fn worker_event_loop<F, Fut>(
 
 	// Run the main worker loop.
 	let rt = Runtime::new().expect("Creates tokio runtime. If this panics the worker will die and the host will detect that and deal with it.");
-	let handle = rt.handle();
 	let err = rt
```

Comment:
Why do we need tokio and futures if everything async-related is already purged? Filesystem interactions are sync in nature, and for the worker reading from the socket is blocking too.

Reply:
You're right! I just didn't remove the rest of async to keep this PR focused. But I can do it here if you want. 

Note that we still need to remove the dependencies on `polkadot-node-core-pvf` and `tracing-gum` to fully remove the dependency on `tokio`.

Reply:
How's tracing-gum related to tokio?

Reply:
I just ran `cargo tree -e normal` in the crate and saw `tokio` several times in the output, e.g. under `sc-network` and `libp2p` crates. I have no idea how `tracing-gum` works though.

Reply:
It's jaeger, even though gum only uses hashing from it.

> You're right! I just didn't remove the rest of async to keep this PR focused. But I can do it here if you want.

Better to polish the rest of the code and properly synchronize threads and take care of removing tokio later (if it's possible)